server/hap/HapProcessCeilometer.cc

```diff
@@ -883,6 +883,33 @@ HatoholError HapProcessCeilometer::fetchItem(const MessagingContext &msgCtx,
 	return err;
 }
 
+HatoholError HapProcessCeilometer::fetchTrigger(const MessagingContext &msgCtx,
```

Comment:
Thank you for comment.
Created a common function, and remove the duplicate lines.